tests/helpers/runif.py

```diff
@@ -64,7 +64,7 @@ def test_wrapper(arg1):
     def __new__(
         self,
         *args,
-        min_gpus: int = 0,
```

Comment:
Don't we want to have many of these `min_gpus` tests still run regardless of whether it's a CUDA GPU or an MPS GPU?

Reply:
I don’t think so. Most of them are either ddp/deepspeed/sharded related or are abput device prsing for the cuda backend